README.md

```diff
@@ -1,3 +1,81 @@
 # loopback-cli
 
 LoopBack CLI tool for creating projects, models and more.
+This package supersedes the older `slc` tool.
+
+## Installation
+
+```
+$ npm install loopback-cli
+```
+
+## Usage
+
+### Getting started
+
+ 1. Run `lb` to create a new LoopBack application
+ 2. Run `node .` to start the scaffolded server
```

Comment:
Nah, it doesn't matter at all since `npm start` in our default scaffold points to `node .` anyways. I was just thinking we should be consistent across the board -- since both are commonly used anyways, this point is probably moot.